Offseason2022/src/main/java/frc/robot/commands/IntakingAction.java

```diff
@@ -4,25 +4,33 @@
 package frc.robot.commands;
 
 import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitUntilCommand;
+import frc.robot.Constants.FCConsts.FCMode;
+import frc.robot.Constants.INConsts.INMode;
+import frc.robot.Constants.TCConsts.TCMode;
+import frc.robot.subsystems.FloorConveyor;
+import frc.robot.subsystems.Intake;
+import frc.robot.subsystems.TowerConveyor;
 
 /**
  *
  */
 public class IntakingAction extends SequentialCommandGroup
 {
-  public IntakingAction( )
+  public IntakingAction(Intake intake, FloorConveyor fConv, TowerConveyor tConv)
   {
+    setName("IntakingAction");
     addCommands(
-    // Add Commands here:
-    // Also add parallel commands using the
-    //
-    // addCommands(
-    // new command1(argsN, subsystem),
-    // parallel(
-    // new command2(argsN, subsystem),
-    // new command3(argsN, subsystem)
-    // )
-    // );
+        // Add Commands here:
+        // Also add parallel commands using the
+        //
+        new IntakeDeploy(true), // Deploy intake

```

Comment:
The comments help, but I think temporarily turning off the formatter like I mentioned in IntakingStop is better. You can still add comments to each line where they make sense, but the comments should really duplicate the names in the code--or they're kind of redundant.